python/mxnet/gluon/loss.py

```diff
@@ -706,3 +706,53 @@ def hybrid_forward(self, F, pred, positive, negative):
                      axis=self._batch_axis, exclude=True)
         loss = F.relu(loss + self._margin)
         return _apply_weighting(F, loss, self._weight, None)
+
+class CosineEmbeddingLoss(Loss):
+    r"""For a target label 1 or -1, vectors target and pred, the function computes the cosine distance
+    between the vectors. This can be interpretted as how similar/dissimilar two input vectors are.
+
+
+    `pred`, `target` can have arbitrary shape as long as they have the same number of elements.
```

Comment:
could you document the formula here? our website supports mathjax so you can write latex for it.

Reply:
@szha Thank you! Incorporating that in the documentation.

Reply:
The rendering needs to be fixed:
http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-12750/2/api/python/gluon/loss.html#mxnet.gluon.loss.CosineEmbeddingLoss
Also, could you list the loss in this file? https://github.com/apache/incubator-mxnet/blame/master/docs/api/python/gluon/loss.md#L14

Reply:
Latest doc: http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-12750/6/api/python/gluon/loss.html#mxnet.gluon.loss.CosineEmbeddingLoss
It requires an empty line after `.. math::` to render properly

Reply:
Please refer to[ huber loss ](https://github.com/gaurav-gireesh/incubator-mxnet/blob/cosineloss/python/mxnet/gluon/loss.py#L478)and use only `{case}`, remove `{gather}`. maybe your mathjax version and what we used in the website is different, so some formula is not recognized. A safe way is to use only what have been correctly rendered, those should be more than enough